.circleci/config.yml

```diff
@@ -119,15 +119,15 @@ commands:
     - run: |
         msbuild \
           -t:Restore \
-          -p:Configuration=Release \
+          -p:Configuration=ReleaseMono \
```

Comment:
Does this purpose to merely have a separated output directory?

Reply:
This is actually for exclusing `Libplanet.Explorer*` projects from being compiled on Mono target. This is implemented via having a new `ReleaseMono` build target on the solution and excluding specific projects from the target, and then using the build target on Mono.

Reply:
Okay, could you mention that in *CONTRIBUTING.md* file?